server/schedule/placement/fit_test.go

```diff
@@ -65,59 +64,59 @@ func (s *testFitSuite) TestFitByLocation(c *C) {
 		{peerStoreID: []uint64{1111, 1112, 1113}, count: 3, expectedPeers: []uint64{1111, 1112, 1113}},
 		{peerStoreID: []uint64{1111, 1112, 1113}, count: 5, expectedPeers: []uint64{1111, 1112, 1113}},
 		// test isolation level
-		{peerStoreID: []uint64{1111}, locationLabels: "zone,rack,host", expectedIsolationLevel: 3},
-		{peerStoreID: []uint64{1111}, locationLabels: "zone,rack", expectedIsolationLevel: 2},
-		{peerStoreID: []uint64{1111}, locationLabels: "zone", expectedIsolationLevel: 1},
-		{peerStoreID: []uint64{1111}, locationLabels: "", expectedIsolationLevel: 0},
-		{peerStoreID: []uint64{1111, 2111}, locationLabels: "zone,rack,host", expectedIsolationLevel: 3},
-		{peerStoreID: []uint64{1111, 2222, 3333}, locationLabels: "zone,rack,host", expectedIsolationLevel: 3},
-		{peerStoreID: []uint64{1111, 1211, 3111}, locationLabels: "zone,rack,host", expectedIsolationLevel: 2},
-		{peerStoreID: []uint64{1111, 1121, 3111}, locationLabels: "zone,rack,host", expectedIsolationLevel: 1},
-		{peerStoreID: []uint64{1111, 1121, 1122}, locationLabels: "zone,rack,host", expectedIsolationLevel: 0},
+		{peerStoreID: []uint64{1111}, locationLabels: "zone,rack,host"},
+		{peerStoreID: []uint64{1111}, locationLabels: "zone,rack"},
+		{peerStoreID: []uint64{1111}, locationLabels: "zone"},
+		{peerStoreID: []uint64{1111}, locationLabels: ""},
+		{peerStoreID: []uint64{1111, 2111}, locationLabels: "zone,rack,host"},
+		{peerStoreID: []uint64{1111, 2222, 3333}, locationLabels: "zone,rack,host"},
+		{peerStoreID: []uint64{1111, 1211, 3111}, locationLabels: "zone,rack,host"},
+		{peerStoreID: []uint64{1111, 1121, 3111}, locationLabels: "zone,rack,host"},
+		{peerStoreID: []uint64{1111, 1121, 1122}, locationLabels: "zone,rack,host"},
```

Comment:
Maybe we can add some checks between these cases to compare their `IsolationScore`?

Reply:
good idea.